song_scrounger/models/album.py

```diff
@@ -0,0 +1,18 @@
+class Album():
+    def __init__(self, name, spotify_uri, artists, popularity=None):
```

Comment:
could you assign a default value of int maybe (-1) to avoid None check below?

Reply:
I'll leave this is in for now because I'm making `popularity` a hard req